src/main/scala-sbt-0.13/compat.scala

```diff
@@ -4,12 +4,21 @@ package sbt
 // Exposes (a minimal part) of the sbt 1 API using sbt 0.13 API
 // Inspired by macro-compat
 
+import scala.language.implicitConversions
+
 object compat {
   val scalaModuleInfo = SettingKey[Option[librarymanagement.ScalaModuleInfo]]("ivyScala")
 
   implicit class ModuleIdOps(val _m: ModuleID) extends AnyVal {
     def withName(n: String): ModuleID = _m copy (name = n)
   }
+
+  implicit class RichProcessBuilder(pb: ProcessBuilder) {
+    def lineStream_!(logger: ProcessLogger) = pb.lines_!(logger)
```

Comment:
Sorry Guillaume , could you also add a little comment explaining that these are here because `lines_!` is deprecated in Scala 2.11+?

Took me a while to go through the history of sbt 0.13/1.0, Scala 2.10/2.11/2.12 to understand why this is here :)

Reply:
Yeah. This is only related to 2.10 vs 2.12. If you use `lines_!` you get a deprecation warning in 2.12, but you can't use `lineStream_!` in 2.10 because it does not exists.